app/static/i18n/fr.json

```diff
@@ -0,0 +1,147 @@
+{
+    "STATUSBAR": {
+        "CLUSTER": "Grappe",
```

Comment:
Je ne suis pas sûre que Grappe soit la bonne traduction ici. Je préfère garder Cluster, c'est beaucoup plus comprehensible.

Reply:
I understand the comment, strictly speaking, `cluster` is an English word exclusively. The official recommendation of the Ministère de la Culture mandates the use of the word `grappe`. It's all up to you 😄 

You can see the details on the official [FranceTerm website](http://www.culture.fr/franceterme/result?francetermeSearchTerme=cluster&francetermeSearchDomaine=0&francetermeSearchSubmit=rechercher&action=search)

Reply:
In the definition of the Ministère de la Culture the definition of grappe is `Ensemble d'appareils de même type (terminaux, ordinateurs, etc.) rattachés à une même unité de contrôle.` Here what we mean by Cluster is more a group of nodes connected to each other not to a control unit.

In any case, I consulted other french speaking developers. I would still keep Cluster and Snapshot even if `Grappe` and `Capture instantanée` are direct translations. It's definitely simpler to understand.